frontend/src/v5/ui/routes/viewer/viewerCanvases.context.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import { createContext, useState } from 'react';
+import { useSearchParam } from '../useSearchParam';
+
+interface ViewerCanvasesContextType {
+	is2DOpen?: boolean;
```

Comment:
is this necessaty now that we are passing !!drawingId,

Reply:
That's to tell components calling useContext, otherwise typescript complains that field doesn't exist, unless you mean something different
![image](https://github.com/3drepo/3drepo.io/assets/43547703/2da42fa7-fcbe-461b-b1ab-57d7a8e78263)